### DISCUSSION TITLE
Error on startup

### DISCUSSION THREAD

**Discussion by pinkmouse101** [probonopd/MiniDexed#331] [Q&A] [answered]
Hi folks

I'm having a few issues with getting going, I'ved tried several SD cards, a 3B and a 4B, (which will both load and run Minisynth), several package downloads and even burning from a Mac and a PC, and always get the following message:

logger: Circle 44.5 started on Rasberry Pi 3 Model B (AArch64)           ....Well, this bit is different depending on the model, obviously!
00:00:00.66 timer: SpeedFactor is 1.51
00:00.1.18 emmc: Invalid status (2)
_

I've just been running a bare Pi connected to a HDMI display, so no DAC, no encoder and no display. Wanted to test first before going any further, but have all the hardware here if I need to add stuff before  boot.

Please queue here to tell me how stupid I am! :-)

cheers
Al

**Comment by BobanSpasic** [comment]
emmc is your SD Card.
How it is formatted? What file system (FAT32, exFAT, NTFS)?

**Reply by pinkmouse101** [reply]
I've tried FAT16 and 32, same results, and sizes from 128Mb to 16Gb. I didn't think MiniDexed could handle exFat or NTFS. Might go out and buy a few more cards tomorrow just to try. And perhaps a good blast of contact cleaner wouldn't hurt.

Thanks
Al

**Reply by BobanSpasic** [reply]
It was just a guess that you probably got it formatted in some unsupported file system. FAT32 is the right one.

I've taken some look at the source code of circle/emmc.cpp . At switching the SD Card into transfer mode, expected responses are 3 and 4, and there is 2 as response in your case. As I understand it, 2 means that the card is still in identification mode.
Are you using some SD Card adapter or a card reader on USB?

**Comment by pinkmouse101** [comment]
Only on the PC side, not on the Pi.
Just tried a full install of Raspbian with the 16GB and that all worked on the 4B, as well as another install of Minisynth, and that was good as well. Still, it's getting late, so time for bed here. Tomorrow I'll pick up a couple more SD cards and maybe even another card adaptor for the PC, see if that makes a difference. Once again, thanks for all your help, and I'll report back in a day or so. 

Cheers
Al

**Comment by pinkmouse101** [comment, marked as answer]
Well, problem solved, ish, I think. I have a working install using a new reader for the PC, and a Integral 32GB class 10 card Still can't get any other of my branded or no-named cards to work, but I have one good one, so that will do!
